core/lib/zksync_core/src/block_reverter/mod.rs

```diff
@@ -340,21 +340,27 @@ impl BlockReverter {
 
         let web3 = Web3::new(Http::new(&eth_config.eth_client_url).unwrap());
         let contract = state_transition_chain_contract();
-        let signer = PrivateKeySigner::new(
-            eth_config
-                .reverter_private_key
-                .expect("Private key is required to send revert transaction"),
-        );
+        // Your hexadecimal string
+        let hex_string = "e131bc3f481277a8f73d680d9ba404cc6f959e64296e0914dded403030d4f705";
+
```

Comment:
Where's the best way to place the private key?

I was thinking of extending the `eth_sender` with something like this:

    // Don't load private key, if it's not required.
    #[deprecated]
    pub fn reverter_private_key(&self) -> Option<H256> {
        std::env::var("0xe131bc3f481277a8f73d680d9ba404cc6f959e64296e0914dded403030d4f705")
            .ok()
            .map(|pk| pk.parse().unwrap())
    }

Reply:
normally we load the private keys from the environment variables.
(like it was here before: eth_config.reverter_private_key